README.md

```diff
@@ -387,8 +394,12 @@ The LTI service must be configured beforehand in A+ by an administrator.
 The `lti` option refers to the label of the LTI service.
 The `url` option may exclude the domain of the service URL since the domain
 must be equal to the URL defined in the LTI service anyway.
-The LTI parameters may be defined in the config.yaml file linked with
-the `config` option, thus it is not required to define them in RST.
+There is two ways to define LTI exercise:
+
+* Use config.yaml file linked with `config` option and define LTI options there.
```

Comment:
using **the**: Use config.yaml file linked with **the** `config` option and define LTI options there. ... in **the** submit directive